src/components/Form.jsx

```diff
@@ -0,0 +1,11 @@
+const Form = () => (
+  <form action="" id="books-form">
+    <label htmlFor="Title">
+      <input type="text" id="title" placeholder="Enter the book title" />
+    </label>
+    <label htmlFor="Author">
+      <input type="text" id="auhor" placeholder="Enter the author's name" />
+    </label>
```

Comment:
- [ ] According to the project requirement, this form should have a button to submit but the button wouldn't add the book *yet*, it should be just a placeholder for now like you did for the remove button. Please fix it in order to make your app work as expected 👍